website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -526,6 +586,58 @@ spec:
           type: 'array<string>',
           description: 'A fixed list of datacenters to try during failover.',
         },
+        {
+          name: 'Targets',
+          type: 'array<ServiceResolverFailoverTarget>',
+          description: `A fixed list of failover targets to try during
+                          failover. It allows operators to express complicated failover
+                          scenarios such as between admin partitions, datacenters and cluster
+                          peers.`,
+          children: [
+            {
+              name: 'Service',
+              type: 'string: ""',
+              description:
+                'The service to use instead of the default of the failover target. If empty the current service is used.',
+            },
+            {
+              name: 'ServiceSubset',
+              type: 'string: ""',
+              description:
+                'The named subset to use for the failover target. If empty the default subset for the requested service is used.',
+            },
+            {
+              name: 'Namespace',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The namespace to use for the failover target. If empty the default namespace is used.',
+            },
+            {
+              name: 'Partition',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                'The admin partition to use for the failover target. If empty the default partition is used.',
```

Comment:
When we say "default partition", do we mean literally the `default` partition? Or it defaults to the partition of this service resolver?

Also, we should provide some guidance on when to use `Partition` vs `Peer`.
```suggestion
                `The admin partition within the same datacenter to use for the failover target.
                If empty, the default partition is used.
                To use an admin partition in a different datacenter for the failover target,
                use the \`Peer\` field instead.`,
```